app/gateway/2.7.x/vitals/vitals-estimates.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Estimate Vitals Storage in PostgreSQL
+badge: enterprise
+---
+
+Vitals data can be divided into two categories: Kong Gateway node statistics and request response codes.
+
+## Kong Gateway node statistics
+
+These types of metrics are proxy latency, upstream latency, and cache hit/miss. Kong Gateway node statistics are stored in tables like the following:
+
+* `vitals_stats_seconds_timestamp` stores 1 new row for every _second_ Kong runs
```

Comment:
```suggestion
* `vitals_stats_seconds_timestamp` stores 1 new row for every _second_ Kong runs.
```
Or, no period in the other two bullet points in this list. For consistency, maybe no periods, as the other lists on the page don't have them (eg lines 20-22)?